### ISSUE TITLE
Save and continue on entry edit shows previous version when static caching turned on

### ISSUE THREAD

**Issue by lukeagsmith** [statamic/v2-hub#1468]
With Static caching turned on using "half measure".

### Expected behaviour
Hitting "Save and continue" on and entry edit screen in the CP should show the version of the content you have just saved.

### Actual behaviour
After the save happens you are jumped to the top of the page, the green save confirmation is displayed but when you scroll down the content is the same as when you first loaded the edit screen.

Refreshing the browser shows the latest saved content.

Turning off static caching solves the problem.

### Steps to reproduce
1.  Turn on static caching
2.  Edit and entry and make changes
3.  Press save and continue
4. See that your edits are not showing
5. Refresh page to see edits

### Server Details

**Statamic Version:**
2.5.10

**Updated from an older Statamic or fresh install:**
Updated

**Comment by jasonvarga** [comment]
Are you sure it has to do with the static caching setting? Can you try it with it disabled?

**Comment by lukeagsmith** [comment]
I did try prior to reporting but I think in my initial tests I turned on "Update the Stache on every request" when I turned off static caching.

I've checked again and this actually appears to be just related to "Update the Stache on every request".

So, regardless of the static cache setting, if "Update the Stache on every request" is off, that is when I see this behaviour.

(Shall I update the issue title?)
